Loop/View Controllers/TextFieldTableViewController.swift

```diff
@@ -85,4 +85,18 @@ extension TextFieldTableViewController {
 
         return vc
     }
+    
+    static func maxIOB(_ value: Double?) -> T {
```

Comment:
The setting VC should have explanatory text to explain how the feature works and that it can be left blank to disable the feature.